capa/features/extractors/ida/helpers.py

```diff
@@ -341,3 +341,20 @@ def find_data_reference_from_insn(insn, max_depth=10):
         ea = data_refs[0]
 
     return ea
+
+
+def get_function_blocks(f):
+    """yield basic blocks contained in specified function
+
+    args:
+        f (IDA func_t)
+    yield:
+        block (IDA BasicBlock)
+    """
+    for block in idaapi.FlowChart(f, flags=(idaapi.FC_PREDS | idaapi.FC_NOEXT)):
+        yield block
```

Comment:
leverage `idaapi.FC_NOEXT` flag to ignore useless external blocks referenced by the function.

from IDA documentation:

```
#define FC_NOEXT   0x0002

do not compute external blocks.

Use this to prevent jumps leaving the function from appearing in the flow chart. Unless specified, the targets of those outgoing jumps will be present in the flow chart under the form of one-instruction blocks
```

Reply:
would you consider including this explanation as a comment on the code?

Reply:
Good idea!